default.nix

```diff
@@ -167,6 +167,35 @@ in rec {
 
   inherit mkAssets;
 
+  dockerImageConfig = args@{exe, name, version ? "latest", created ? "now",extraContents ? [], extraPaths ? []}:
+    let
+      appDirSetupScript = nixpkgs.runCommand "appDirSetupScript.sh" {} ''
+        mkdir -p    $out/var/lib/backend
+        ln -sft $out/var/lib/backend '${exe}'/*
+        ${nixpkgs.findutils}/bin/find $out/var/lib/backend
+        '';
+    in {
+      name = name;
+      tag = version;
+      created = created;
+      contents = [ nixpkgs.iana-etc nixpkgs.cacert appDirSetupScript ] ++ extraContents;
+      config = {
+        Env = [
+          ("PATH=" + builtins.concatStringsSep(":")(extraPaths ++ [
+            "/var/lib/backend" # put the obelisk project on the path.
+            "/bin" # put contents on path
+          ] ++ map (pkg: "${pkg}/bin") pkgs.stdenv.initialPath # put common tools in path so docker exec is useful
+          ))
+        ];
+        Expose = 8000;
+        Entrypoint = ["/var/lib/backend/backend"];
```

Comment:
We shouldn't have hardcoded this. We'll probably have to move this nix into the `mkObelisk` module function so that one can get a docker image for obelisk without assuming default configuration.

Reply:
Will look into it

Reply:
If I understood this correctly then any changes on internalPort should propagete to the docker target. It should now not be hardcoded.

Reply:
As per the entrypoint, if I understand this correctly that location is being grabbed from nix and symlinked in line 172

Reply:
Hi @3noch, do the most recent version address this concern?